cmds/ocm/commands/ocmcmds/common/addhdlrs/options.go

```diff
@@ -8,30 +8,51 @@ import (
 )
 
 type Options struct {
-	Replace bool
+	Replace           bool
+	PreserveSignature bool
 }
 
-var _ ocm.ModificationOption = (*Options)(nil)
+var (
+	_ ocm.ModificationOption        = (*Options)(nil)
+	_ ocm.ElementModificationOption = (*Options)(nil)
+	_ ocm.BlobModificationOption    = (*Options)(nil)
+	_ ocm.TargetElementOption       = (*Options)(nil)
+)
 
 func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	f := fs.Lookup("replace")
-	if f != nil {
-		if bp := generics.Cast[*bool](f.Value); bp != nil {
-			return
-		}
+	if f == nil {
+		fs.BoolVarP(&o.Replace, "replace", "R", false, "replace existing elements")
+	}
+
+	f = fs.Lookup("preserve-signature")
```

Comment:
why do you use fs.Lookup before assigning variable? (in which case would this already be set?)

Reply:
the same optio set might be used multiple times, then the option can only be added once.